packages/fxa-payments-server/src/components/DialogMessage.tsx

```diff
@@ -20,10 +21,21 @@ export const DialogMessage = ({
   const dialogInsideRef = useClickOutsideEffect<HTMLDivElement>(onDismiss);
   return (
     <Portal id="dialogs">
-      <div data-testid="dialog-message-container" className={classNames('blocker', 'current')}>
-        <div data-testid="dialog-message-content" className={classNames('modal', className)} ref={dialogInsideRef}>
-          <button data-testid="dialog-dismiss" className="dismiss" onClick={onDismiss as () => void}>
-            &#x2715;
+      <div
+        data-testid="dialog-message-container"
+        className={classNames('blocker', 'current')}
+      >
+        <div
+          data-testid="dialog-message-content"
+          className={classNames('modal', className)}
+          ref={dialogInsideRef}
+        >
+          <button
+            data-testid="dialog-dismiss"
+            className="dismiss"
+            onClick={onDismiss as () => void}
```

Comment:
Can we add an `aria-label="Close modal"` on here? Looks good otherwise!

FWIW, I have an accessibility epic open for payments and haven't had a chance yet to finish an audit, if you want to get this merged (per your comment in #3187) will you create an issue and add it to #2963? Thanks!

Reply:
👍  Add #3227 to the accessibility epic.